src/main/scala/loamstream/compiler/LoamCompiler.scala

```diff
@@ -119,16 +128,16 @@ object LoamCompiler extends Loggable {
         val soManyErrors = soMany(errors.size, "error")
         val soManyWarnings = soMany(warnings.size, "warning")
         val soManyInfos = soMany(infos.size, "info")
-        
+
         s"There were $soManyErrors, $soManyWarnings, $soManyInfos."
       }
     }
-    
+
     final case class FailureDueToException(
-        errors: Seq[Issue],
-        warnings: Seq[Issue],
-        infos: Seq[Issue],
-        throwable: Throwable) extends Result {
+      errors: Seq[Issue],
+      warnings: Seq[Issue],
+      infos: Seq[Issue],
+      throwable: Throwable) extends Result {
```

Comment:
Wouldn't this just be `extends Failure` instead of `Result` and simply add `throwable` to it?

Reply:
Perhaps, but it's either not possible, or not advisable to extend a `case class` with another `case class` (or maybe both).  The common parts could be factored out, but that's more trouble than it's worth in this case, when the subtypes of `Result` are mostly just containers for data.  This is very old code that's been working forever; it only shows up in the diff due to whitespace changes.